src/tests/test_e2e_install.py

```diff
@@ -37,6 +38,47 @@ def test_olm_operator(self, cluster, olm_operators):
         cluster.prepare_for_installation()
         cluster.start_install_and_wait_for_installed(fall_on_pending_status=True)
 
+    @JunitTestSuite()
+    def test_mce_storage_post(self):
+        config.load_kube_config()
+
+        # get agent service config
+        asc_list = client.CustomObjectsApi().list_cluster_custom_object(
+            group="agent-install.openshift.io", version="v1beta1", plural="agentserviceconfigs"
+        )
+        assert "items" in asc_list
+        assert len(asc_list["items"]) == 1, "Expected to have only one agentserviceconfig resource"
+        agent_service_config = asc_list["items"][0]
+
+        # get storage classes
+        storage_classes_list = client.StorageV1Api().list_storage_class()
+        assert len(storage_classes_list.items) > 0, "Expected storage class list to have one item or more"
+        storage_classes = [storage_class.metadata.name for storage_class in storage_classes_list.items]
+
+        # each storage pvc should have an existing storage class
+        storageClaims = ["databaseStorage", "filesystemStorage", "imageStorage"]
+        for key in storageClaims:
+            assert (
+                agent_service_config["spec"][key]["storageClassName"] in storage_classes
+            ), f"Expected {key} to match existing storageclass"
+
+        v1 = client.CoreV1Api()
+        mce_namespace = "multicluster-engine"
+        # pvc should be bound
+        pvcs = v1.list_namespaced_persistent_volume_claim(mce_namespace)
+        for pvc in pvcs.items:
+            assert pvc.status.phase == "Bound", "Expected pvc to be bound"
+
+        # workloads should be running
+        selectors = ["app=assisted-service", "app=assisted-image-service"]
+        for selector in selectors:
+            pods = v1.list_namespaced_pod(mce_namespace, label_selector=selector)
+            pod = pods.items[0]
+            for status in pod.status.container_statuses:
+                assert status.ready, "Expected pod to be ready"
+                assert status.started, "Expected pod to be started"
+                assert status.state.running is not None, "Expected pod to be running"
+
```

Comment:
Does the pod have enough time to be ready ? maybe we need some kind of waiting ?

Reply:
Good point, we might need some waiting here